modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_getting_started/quickstart.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-creating-account-wide-sts-roles-and-policies_{context}"]
+= Creating the account-wide STS roles and policies
+
+Before using the {cluster-manager-first} {hybrid-console-second} to create {hcp} for {product-title} (ROSA) clusters, create the required account-wide roles and policies, including the Operator policies.
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for {hcp} for ROSA with STS.
+* You have available AWS service quotas.
```

Comment:
Service quota for what?  There are other statements like this, are they all the same or does it vary depending on the context of the page?  Unclear as a customer how to check if this pre-req is met.

Reply:
There should already be a ROSA service quotas doc...https://docs.openshift.com/rosa/rosa_planning/rosa-sts-required-aws-service-quotas.html